### DISCUSSION TITLE
Is okay to use PubSub for 1-1 chat? {Question}

### DISCUSSION THREAD

**Discussion by NuniTelo** [uNetworking/uWebSockets.js#472] [Q&A] [answered]
Hi everyone,
I implemented uWebsocket  to my current project and I was thinking if using pubsub is the way to go with 1-1 chat application.
Thank you.

**Comment by ghost** [comment, marked as answer]
No that's not a good idea. But it depends on your requirements. It's ultimately up to you to decide. Pub/sub helps you solve one-to-many problems and will have a certain overhead for 1-to-1 (especially with SHARED_COMPRESSOR).

**Comment by NuniTelo** [comment]
Thank you so much for your response. Please do you have any example for implementing 1-1 ?

**Comment by ghost** [comment]
1-to-1 is just regular ws.send just like you have in the web browser.
